packages/core/src/definitions/styles.ts

```diff
@@ -164,26 +165,6 @@ export const builtInStyles: Partial<
     },
     description: 'hyperlink for section or container',
   },
-  cfImageFormat: {
```

Comment:
how come these went away?

Reply:
All the props in `builtInStyles` get added to all component definitions via `enrichComponentDefinition()` calling `applyBuiltInStyleDefinitions()`

![Screenshot 2024-02-28 at 3 50 40 PM](https://github.com/contentful/experience-builder/assets/8539634/4626e418-da71-4862-9f22-3d796233bcc4)

With these props removed from `builtInStyles` we will no longer add these variables to all component definitions:

![Screenshot 2024-02-28 at 3 52 37 PM](https://github.com/contentful/experience-builder/assets/8539634/0b4d05e4-cad7-4bf5-9e19-1328c3f9485a)

When the props are in `optionalBuiltInStyles`, they will only be added to components that specifically include them in the component definition array of `builtInStyles`

![Screenshot 2024-02-28 at 3 56 11 PM](https://github.com/contentful/experience-builder/assets/8539634/4f35aa5e-fcf2-4418-aa6c-3822dfc353f5)